regimes/it/tags.go

```diff
@@ -0,0 +1,339 @@
+	TagDeclarationOfIntent        cbc.Key = "declaration-of-intent"
+	TagScrap                      cbc.Key = "scrap"
+	TagPreciousMetals             cbc.Key = "precious-metals"
+	TagConstructionSubcontracting cbc.Key = "construction-subcontracting"
+	TagBuildings                  cbc.Key = "buildings"
+	TagMobile                     cbc.Key = "mobile"
+	TagElectronics                cbc.Key = "electronics"
+	TagConstruction               cbc.Key = "construction"
+	TagEnergy                     cbc.Key = "energy"
+)
+
+// This is only a partial list of all the potential tags that
+// could be available for use in Italy. Given the complexity
+// involved, we've focussed here on the most useful.
+var invoiceTags = []*tax.Tag{
+	// *** Document Type Tags ***
+	{
+		Key: TagFreelance,
+		Name: i18n.String{
+			i18n.EN: "Freelancer",
+			i18n.IT: "Parcella",
+		},
+	},
+	{
+		Key: common.TagReverseCharge,
+		Name: i18n.String{
+			i18n.EN: "Reverse Charge",
+			i18n.IT: "Inversione del soggetto passivo",
+		},
+	},
+	{
+		Key: common.TagSelfBilled,
+		Name: i18n.String{
+			i18n.EN: "Self-billed",
+			i18n.IT: "Autofattura",
+		},
+	},
+	{
+		Key: TagCeilingExceeded,
+		Name: i18n.String{
+			i18n.EN: "Ceiling exceeded",
+			i18n.IT: "Splafonamento",
+		},
+	},
+	{
+		Key: TagSanMerinoPaper,
+		Name: i18n.String{
+			i18n.EN: "Purchases from San Merino with VAT and paper invoice",
+			i18n.IT: "Acquisti da San Merino con IVA e fattura cartacea",
+		},
+	},
+
+	// **** Fiscal Regime Tags ****
+	{
+		Key: TagMinimumTaxPayers,
+		Name: i18n.String{
+			i18n.EN: "Minimum Taxpayers",
+			i18n.IT: "Contribuenti minimi",
+		},
+	},
+}
+
+var vatZeroTaxTags = []*tax.Tag{
+	{
+		Key: TagExcluded,
+		Name: i18n.String{
+			i18n.EN: "Excluded pursuant to Art. 15, DPR 633/72",
+			i18n.IT: "Escluse ex. art. 15 del D.P.R. 633/1972",
+		},
+		Meta: cbc.Meta{
+			KeyFatturaPANatura: "N1",
+		},
+	},
+	{
+		Key: TagNotSubject,
+		Name: i18n.String{
+			i18n.EN: "Not subject (this code is no longer permitted to use on invoices emitted from 1 January 2021)",
+			i18n.IT: "Non soggette (questo codice non è più utilizzabile a partire dal 1° gennaio 2021)",
+		},
+		Meta: cbc.Meta{
+			KeyFatturaPANatura: "N2",
+		},
+	},
+	{
+		Key: TagNotSubject.With(TagArticle7),
+		Name: i18n.String{
+			i18n.EN: "Not subject pursuant to Art. 7, DPR 633/72",
+			i18n.IT: "Non soggette ex. art. 7 del D.P.R. 633/72",
+		},
+		Meta: cbc.Meta{
+			KeyFatturaPANatura: "N2.1",
+		},
+	},
+	{
+		Key: TagNotSubject.With(TagOther),
+		Name: i18n.String{
+			i18n.EN: "Not subject - other",
+			i18n.IT: "Non soggette - altri casi",
+		},
+		Meta: cbc.Meta{
+			KeyFatturaPANatura: "N2.2",
+		},
+	},
+	{
+		Key: TagNotTaxable,
+		Name: i18n.String{
+			i18n.EN: "Not taxable (this code is no longer permitted to use on invoices emitted from 1 January 2021)",
+			i18n.IT: "Non imponibili (questo codice non è più utilizzabile a partire dal 1° gennaio 2021)",
```

Comment:
I feel like these are descriptions rather than names. I liked how `Scheme`s had key, name, and description separately. `Key` was useful programmatically while the shorthand `Name` and a detailed `Description` provided value for end-users.